tests/unit/moduleapi/rdbloadsave.tcl

```diff
@@ -80,6 +80,9 @@ start_server {tags {"modules"}} {
             fail "Can't find 'Killing AOF child' in recent log lines"
         }
 
+        # Verify that we have enabled the AOF
+        assert_equal 1 [s aof_enabled]
```

Comment:
If this is `server.aof_enabled`, then it would never change since we never change `server.aof_enabled`, right?

What exactly is this INFO field showing?

Reply:
this field is actually `server.aof_state != AOF_OFF`, i guess the showing is the aof is active or not.
```
            "aof_enabled:%d\r\n", server.aof_state != AOF_OFF,
```

Reply:
OK, great. So maybe change the comment here in the test case?

    # Verify that AOF is active.

Reply:
>this field is actually server.aof_state != AOF_OFF

This is really confusing ...